packages/legend-studio/src/stores/editor-state/element-editor-state/service/ServiceRegistrationState.ts

```diff
@@ -217,19 +231,42 @@ export class ServiceRegistrationState {
           versionInput,
         )) as ServiceRegistrationResult;
       if (this.activatePostRegistration) {
+        this.setRegistrationPhase(
+          SERVICE_REGISTRATION_PHASE.ACTIVATING_SERVICE,
+        );
         yield this.editorStore.graphManagerState.graphManager.activateService(
           serverUrl,
           serviceRegistrationResult.serviceInstanceId,
         );
       }
-      this.setShowModal(false);
-      this.editorStore.applicationStore.notifySuccess(
-        `service with patten ${serviceRegistrationResult.pattern} registered ${
-          this.activatePostRegistration ? ' and activated ' : ''
-        } on ${serviceRegistrationResult.serverURL}`,
-        undefined,
-        null,
-      );
+      const message = `Service with patten ${
+        serviceRegistrationResult.pattern
+      } registered ${this.activatePostRegistration ? 'and activated ' : ''}on ${
+        serviceRegistrationResult.serverURL + serviceRegistrationResult.pattern
+      }.`;
+      this.editorStore.setActionAltertInfo({
+        message,
+        prompt: 'You can open the corresponding service page',
```

Comment:
--> `You can now launch and monitor the operation of your service`

Reply:
Speaking of which, would be nice if we could go directly to `Legend Services`, we could add another config for Legend Services URL. What do you think?